### PR TITLE
Remove non-working android specific code for scrollview

### PR DESCRIPTION
**Pull request by basvdheijden**
See https://github.com/leecade/react-native-swiper/issues/449

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -545,29 +545,18 @@ export default class extends Component {
   }
 
   renderScrollView = pages => {
-    if (Platform.OS === 'ios') {
```
Comment (xxxMortxxx) [inline src/index.js]: Awesome work dude). I wanted to create same pull request, and I really need this to be merged)
But you also need to remove code for android at line 396, or you will get an error.

Comment (basvdheijden) [inline src/index.js]: Done.

**Comment by arribbar** [comment]
I'll take a look at this PR. Thank you.
You might want to fix conflicts in order to merge it later on

**Comment by basvdheijden** [comment]
I've resolved the conflicts.

**Comment by arribbar** [comment]
@basvdheijden Thank you :)
Can you explain a bit more what you are doing here and how we can test it, maybe with an example?

**Comment by AtticusFetch** [comment]
@arribbar I believe this is related to [this issue](https://github.com/facebook/react-native/issues/13463#issuecomment-310896400).
We also faced a problem with ViewPagerAndroid not showing dynamically added children, but a workaround with ```key={pages.length}``` seems to do the thing (It works locally).

**Comment by arribbar** [comment]
Ok, thanks @AtticusFetch. 
@basvdheijden Can you make the change and remove setDimension function ? Or @AtticusFetch, you can make another PR, up to you guys ! :)

**Comment by AtticusFetch** [comment]
I would really like it merged ASAP, so, @basvdheijden, hope you won't mind if I create another PR with ```key={pages.length}``` fix

**Comment by arribbar** [comment]
Thanks for your work @AtticusFetch and above all, @basvdheijden. I close this PR now the other one is merged

**Pull request closed without merging**